framework/integration/test_resource_set/test_resource/resource.go

```diff
@@ -0,0 +1,46 @@
+// +build k8srequired
+
+package testresource
```

Comment:
In go it is common to name the directory like the package to have a cleaner and more straight forward import. This is removing the underscores from the directory names.

Reply:
I renamed the folders.